### PR TITLE
Only check formatting on modified files

### PR DESCRIPTION
**Pull request by Dragorn421**

### CODE REVIEW

**Comment by Dragorn421** [comment]
Reafy

**Review by fig02 — APPROVED** [review]

File: format.py [hunk 1]
```diff
@@ -175,10 +185,18 @@ def main():
...
-        files = args.files
+    if args.files or args.files_list:
+        files = []
+        files.extend(args.files)
+        if args.files_list:
+            files.extend(Path(args.files_list).read_text().split())
+        files = list(files)
         extra_files = []
+        if args.verbose:
+            print("Formatting specific files:", len(files), files)
```
Comment (cadmic) [inline format.py:196]: Nit: I think something like `f"Formatting {len(files)} files: ' '.join(files)"` would be more friendly, but I doubt many people will actually read this

**Review by cadmic — APPROVED** [review]

File: format.py (hunk 1 above)
Comment (Dragorn421) [reply format.py:196]: yeah this is meant for debugging when this implementation explodes, I don't really care how it looks

**Review by hensldm — APPROVED** [review]
Neat. Useful for MM as well :) .

**Pull request merged**